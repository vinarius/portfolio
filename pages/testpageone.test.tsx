import React from 'react';
import renderer from 'react-test-renderer';

import TestPageOne from './testpageone';

describe('<TestPageOne />', ()=>{
  it('should render a p element with text in it', ()=>{
    const component = renderer.create(
      <TestPageOne />
    ).toJSON();

    expect(component).toMatchSnapshot();
  });
});