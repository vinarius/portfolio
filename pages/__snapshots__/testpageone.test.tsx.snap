// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TestPageOne /> should render a p element with text in it 1`] = `
<p>
  Test page one is working
</p>
`;
